src/Stryker.Core/Stryker.Core/TestRunners/VsTest/DiscoveryEventHandler.cs

```diff
@@ -11,6 +11,7 @@ public class DiscoveryEventHandler : ITestDiscoveryEventsHandler
         private AutoResetEvent waitHandle;
         private readonly List<string> _messages;
         public List<TestCase> DiscoveredTestCases { get; private set; }
+        public bool Aborted { get; private set; }
```

Comment:
This is just for capturing what VsTest reports. There is now specific documentation explaining why it could be aborted. I though it could be useful for debugging

Reply:
discovery could be aborted if test run fails or is externally killed for some reason. As of now, it logs the event for analysis if issue reoccurs. But there is no reason to ask for an abort